### ISSUE TITLE
HD Tags for Movies?

### ISSUE THREAD

**Issue by GititGititHub81** [griff/metaz#148]
Great app!  It works beautifully!  I just need one more thing and it's perfect and that would be the ability to add (HD) tagging to your movies.  

Just out of curiosity, is this a tough issue to tackle?  It must be since it hasn't made it in yet..

Anyway, thank you so much for your work on this awesome app!

**Comment by MORGiON666** [comment]
Hi, I agree this feature would make the app perfect.

**Comment by saheljalal** [comment]
Yeah seriously, this is like a pretty important feature.  Any updates on it?

**Comment by WunjoMan** [comment]
This is a big feature for me, and something that was present with iDentify app that I previously used.  Identify went bye-bye with Sierra.  I like a lot of the Features in Meta Z better.  A very nice app.
